skbio/stats/distance/tests/test_base.py

```diff
@@ -467,6 +468,32 @@ def test_to_data_frame_default_ids(self):
                            index=['0', '1'], columns=['0', '1'])
         assert_data_frame_almost_equal(df, exp)
 
+    def test_to_series_1x1(self):
+        series = self.dm_1x1.to_series()
+        exp = pd.Series([0.0],
+                        pd.MultiIndex(levels=[['a'], ['a']],
+                        labels=[[0], [0]]))
+        exp = exp[exp > 0]
```

Comment:
The expected value should be `pd.Series([], index=[])` for a 1x1 matrix. I think you're obtaining that on this line but it's better to explicitly define the expected value so that it's clear what you're comparing against.